ember-core/shared/src/main/scala/org/http4s/ember/core/Parser.scala

```diff
@@ -373,6 +373,12 @@ private[ember] object Parser {
     def parser[F[_]: Concurrent](maxHeaderSize: Int)(
         buffer: Array[Byte],
         read: Read[F],
+    ): F[(Response[F], Drain[F])] = parser2[F](maxHeaderSize)(buffer, read, discardBody = false)
+
+    def parser2[F[_]: Concurrent](maxHeaderSize: Int)(
```

Comment:
Just curious, won't the method overloading work here?

Reply:
it will work, i had to move the discardBody parameter to the first parameter list